src/pages/UIComponentsPage.js

```diff
@@ -8,25 +8,94 @@ import { ReactComponent as MapIcon } from '../assets/map-icon.svg';
 import { ReactComponent as SearchPlaceIcon } from '../assets/search-place-icon.svg';
 import { ReactComponent as ShareLocationIcon } from '../assets/share-location-icon.svg';
 import { ReactComponent as UserLocationIcon } from '../assets/user-location-icon.svg';
+import { ReactComponent as SettingsIcon } from '../assets/settings-icon.svg';
+import { ReactComponent as HelpIcon } from '../assets/help-icon.svg';
+import { Button } from '../components/Button';
 
 export const UIComponentsPage = () => {
   return (
-    <div>
+    <div className="view__content">
       <h1>UI Components</h1>
       <section>
         <h2>Iconography</h2>
         <br />
         <div className="icons">
-          <CarIcon className="icon" />
-          <TrainIcon className="icon" />
-          <BicycleIcon className="icon" />
-          <CloseIcon className="icon" />
-          <FlyToMeIcon className="icon" />
-          <ListIcon className="icon" />
-          <MapIcon className="icon" />
-          <SearchPlaceIcon className="icon" />
-          <ShareLocationIcon className="icon" />
-          <UserLocationIcon className="icon" />
+          <CarIcon />
+          <TrainIcon />
+          <BicycleIcon />
+          <CloseIcon />
+          <FlyToMeIcon />
+          <ListIcon />
+          <MapIcon />
+          <SearchPlaceIcon />
+          <ShareLocationIcon />
+          <UserLocationIcon />
+          <SettingsIcon />
+          <HelpIcon />
+        </div>
+      </section>
+      <br />
+      <section>
+        <h2>Icons (as Buttons)</h2>
+        <br />
+        <div className="icons">
+          <Button
+            className="btn btn__icon"
+            label="Share location button"
```

Comment:
Minor change for this and the next few labels- to remove the word "button". The way it currently reads with VoiceOver, it will say the word "button" twice since it is already wrapped with `<Button`.